packages/shorebird_cli/lib/src/shorebird_flutter.dart

```diff
@@ -62,7 +71,24 @@ class ShorebirdFlutter {
       rethrow;
     }
 
-    installProgress.complete();
+    final precacheProgress = logger.progress(
+      'Running ${lightCyan.wrap('flutter precache')}',
+    );
+
+    try {
+      await process.run(
+        executable,
+        ['precache', ...precacheArgs],
+        workingDirectory: targetDirectory.path,
+        runInShell: true,
+      );
+      precacheProgress.complete();
+    } catch (_) {
+      precacheProgress.fail('Failed to precache Flutter $version');
+      logger.info(
+        '''This is not a critical error, but your next build make take longer than usual.''',
+      );
```

Comment:
minor but maybe we can consolidate these into a single log like:

precacheProgress.fail('Failed to precache Flutter $version -- the next build make take longer than usual. ');

Reply:
I wanted to have a part of the message that wasn't a "failure" or "error" message to head off support requests when/if this fails (just to be extra clear that the thing that went wrong isn't going to cause shorebird to not work)